src/wasm/wasm-emscripten.cpp

```diff
@@ -479,7 +479,11 @@ std::string EmscriptenGlueGenerator::generateEmscriptenMetadata() {
     meta << "\n  },\n";
   }
 
-  meta << "  \"tableSize\": " << wasm.table.initial.addr << ",\n";
+  if (!wasm.tables.empty()) {
+    meta << "  \"tableSize\": " << wasm.tables[0]->initial.addr << ",\n";
+  } else {
+    meta << "  \"tableSize\": 0,\n";
+  }
```

Comment:
@kripken would it make more sense to error out here for now when there are multiple tables, or would this metadata item only care about table 0 anyway?

Reply:
ping @kripken, or maybe @sbc100 knows?

Reply:
I think this should be fine, and if it's not, we can address that once LLVM supports multiple tables.